core/lib/basic_types/src/protocol_version.rs

```diff
@@ -297,6 +297,17 @@ pub struct ProtocolSemanticVersion {
 impl ProtocolSemanticVersion {
     const MAJOR_VERSION: u8 = 0;
 
+    pub fn new(minor: ProtocolVersionId, patch: VersionPatch) -> Self {
+        Self { minor, patch }
+    }
+
+    pub fn current_prover_version() -> Self {
```

Comment:
UPD: I will move this into prover workspace after https://github.com/matter-labs/zksync-era/pull/2062 will be merged, since we have dependencies in core workspace on this method right now. Linear ticket and comments added